web_interface/blueprints/api_v3.py

```
@@ -6903,4 +6974,678 @@ def clear_old_errors():
+        )
+
+
+# ─── Starlark Apps API ──────────────────────────────────────────────────────
+
+def _get_tronbyte_repository_class():
+    """Import TronbyteRepository from plugin-repos directory."""
+    import importlib.util
+
+    module_path = PROJECT_ROOT / 'plugin-repos' / 'starlark-apps' / 'tronbyte_repository.py'
+    if not module_path.exists():
+        raise ImportError(f"TronbyteRepository module not found at {module_path}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)